digdag-docs/src/architecture.md

```diff
@@ -53,6 +53,16 @@ You can use [Docker](https://www.docker.com/) to run tasks in a container.  If `
     +step1:
       py>: tasks.MyWorkflow.step1
 
+Digdag caches pulled image and reuses it. By default, currently, Digdag uses the cached image consistently even if there is an update. You can set `pull_always: true` option so that Digdag checks update of the image every time when a task starts.
```

Comment:
> You can set `pull_always: true` option so that Digdag checks update of the image every time when a task starts

Strictly speaking, this doesn't say that Digdag pulls an update image after checking that if it exists. Adding that might be better.